image_width: 1440
image_height: 1080
camera_name: hk_camera_right
camera_matrix:
  rows: 3
  cols: 3
  data: [ 1802.98196,    0.     ,  722.4003 ,
          0.     , 1801.2397 ,  533.82024,
          0.     ,    0.     ,    1. ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [ -0.077961, 0.090236, -0.000110, -0.000304, 0.000000 ]
rectification_matrix:
  rows: 3
  cols: 3
  data: [ 1., 0., 0.,
          0., 1., 0.,
          0., 0., 1. ]
projection_matrix:
  rows: 3
  cols: 4
  data: [ 1783.17188,    0.     ,  721.66578,    0.     ,
          0.     , 1788.14014,  533.20097,    0.     ,
          0.     ,    0.     ,    1.     ,    0. ]
extrinsic:
  rows: 3
  cols: 4
  data: [ -0.23025,  -0.973131,  0.00111945,  -0.193204,
          0.0154222,  -0.00479921,  -0.99987,  0.0518421,
          0.973009,  -0.230203,  0.0161128,  -0.0234456 ]
extrinsic2:
  rows: 3
  cols: 4
  data: [ -0.207801,  -0.978168,  0.00228503,  -0.191765,
          -0.0161436,  0.0010938,  -0.999869,  0.0339776,
          0.978038,  -0.207811,  -0.0160184,  -0.00389894 ]
world_to_cam:
  rows: 3
  cols: 4
  data: [0.9620551806944498, 0.2690649240702225, -0.04531992866338747, -14.3086982404946,
    0.02403825232296894, 0.08187066028518783, 0.9963530285043171, 1.26029623711392,
    0.2717940244453995, -0.9596360047536526, 0.07229624233817983, 27.33105752799093]

